framework/gui.h

```diff
@@ -470,6 +472,9 @@ class Gui
 	/// Scale factor to apply to the size of gui elements (expressed in dp)
 	float dpi_factor{1.0f};
 
+	///  Scale factor to apply due to a difference between the window and GL pixel sizes
+	float retina_factor{1.0f};
```

Comment:
Rename to `x_content_scale_factor`

Reply:
changed to `content_scale_facter` per above.